src/lib/viewers/doc/DocPreloader.js

```diff
@@ -101,17 +100,18 @@ class DocPreloader extends EventEmitter {
                 this.wrapperEl.className = this.wrapperClassName;
                 this.wrapperEl.innerHTML = `
                 <div class="${CLASS_BOX_PREVIEW_PRELOAD} ${CLASS_INVISIBLE}">
-                    <img class="${CLASS_BOX_PREVIEW_PRELOAD_CONTENT}" src="${this.srcUrl}" />
-                    <div class="${CLASS_BOX_PREVIEW_PRELOAD_CONTENT} ${CLASS_BOX_PREVIEW_PRELOAD_OVERLAY}">
-                        ${SPINNER_HTML}
+                    <div class="${CLASS_BOX_PREVIEW_PRELOAD_PLACEHOLDER}">
+                        <div class="${CLASS_BOX_PREVIEW_PRELOAD_BACKGROUND}"></div>
+                        <img class="${CLASS_BOX_PREVIEW_PRELOAD_CONTENT}" src="${this.srcUrl}" />
+                        <div class="${CLASS_BOX_PREVIEW_PRELOAD_OVERLAY}"></div>
```

Comment:
Do we still need the overlay element?

Reply:
Yep, it's still called for in the new design.